src/WorkOS.net/Services/UserManagement/Entities/Reasons.cs

```diff
@@ -0,0 +1,24 @@
+namespace WorkOS
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using Newtonsoft.Json;
+
+    /// <summary>
+    /// Reasons for unauthorized access to an organization the user is a member of.
+    /// </summary>
+    public class Reasons
```

Comment:
Seems like this should be named in the singular?

```suggestion
    public class Reason
```

Reply:
Agree, I was just following doc naming but realized the entities can remain singular 
<img width="408" alt="CleanShot 2023-08-21 at 12 02 33@2x" src="https://github.com/workos/workos-dotnet/assets/5375567/64fcb42e-9669-4cc3-b2ba-92ba7ea121e9">